src/main/java/pl/betse/beontime/utils/CustomResponseMessage.java

```diff
@@ -0,0 +1,24 @@
+package pl.betse.beontime.utils;
+
+
+import lombok.*;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CustomResponseMessage {

```

Comment:
we don't want to have any custom response, especially with fields that are already in HTTP Message